molecule/cookiecutter/scenario/driver/vagrant/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/prepare.yml

```diff
@@ -1,11 +0,0 @@
----
```

Comment:
Why is this deleted? I thought we needed this in the cookiecutter folder?

Reply:
The suggested change is that molecule will use a built-in playbook for the `prepare` step on `vagrant` instead of an explicitly rendered (default) playbook.